docker-compose.yml

```diff
@@ -1,6 +1,9 @@
 proxy:
-  image: kuzzleio/proxy:alpine
+############# UPDATE TO kuzzleio/proxy:alpine
+############# WHEN RELEASING RC5
+  image: kuzzleio/proxy:develop
   ports:
+    - "5713:5713"
```

Comment:
It might be better to align this port to map 7513 instead of 5713; it can be confusing and error pron with the other non-standard ports (7511 & 7512)